src/components/LoggedInFeature/LoginRequiredModal.messages.js

```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  featureBlockedTitle: {
+    id: 'cboard.components.LoginRequiredModal.featureBlockedTitle',
+    defaultMessage: 'This feature is blocked'
+  },
+  featureBlockedText: {
+    id: 'cboard.components.LoginRequiredModal.featureBlockedText',
+    defaultMessage:
+      'Cboard disabled this feature. To continue using it please sign up or login'
```

Comment:
please replace this text with something similar than:
'This feature is disabled for unlogged users. To continue using it please sign up or login'